### PR TITLE
#20 Prompt "are you still on duty?" 10 hours after going on duty

### LINKED ISSUE #20: Prompt "are you still on duty?" 10 hours after going on duty

### PR DESCRIPTION
**Pull request by VitaliyGre**
Fixed #20

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -217,4 +217,8 @@
     <string name="continue_with_empty_fields">Continue with empty fields?</string>
     <string name="continue_action">Continue</string>
     <string name="chose_image_source">Choose image source</string>
+
+    <!--    Notifications-->
+    <string name="channel_name">Channel name</string>
```
Comment (lebedynskyi) [inline app/src/main/res/values/strings.xml:222]: Could it be  :
-  O-Fish notifications
-  O-Fish will show on duty reminder each 10 hours?

File: app/src/main/java/org/wildaid/ofish/ui/home/TimerReceiver.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import org.wildaid.ofish.R
+import org.wildaid.ofish.app.CHANNEL_ID
+
+private const val notificationId = 119
+
+class TimerReceiver : BroadcastReceiver() {
```
Comment (lebedynskyi) [inline app/src/main/java/org/wildaid/ofish/ui/home/TimerReceiver.kt:13]: Could it be something like NotificationAlarmSender?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (VitaliyGre) [reply app/src/main/res/values/strings.xml]: Done

File: app/src/main/java/org/wildaid/ofish/ui/home/TimerReceiver.kt (hunk 2 above)
Comment (VitaliyGre) [reply app/src/main/java/org/wildaid/ofish/ui/home/TimerReceiver.kt]: Done

**Pull request merged**